libs/core/activity/IActivitySequencer.hpp

```diff
@@ -85,6 +85,19 @@ class ACTIVITY_CLASS_API IActivitySequencer : public IActivityLauncher
         const data::Composite::csptr& overrides = nullptr
     );
 
+    /**
+     * @brief Remove the activity with index and its requirements and all the following activities.
+     *
+     * This is used to clear the series and requirement when going backward.
+     *
+     * @param seriesDB Series DB containing all the activities
+     * @param index the activity in index and all the following will be removed
+     */
+    ACTIVITY_API void clearLastActivities(
+        const data::SeriesDB::sptr& seriesDB,
+        size_t index
```

Comment:
Prefer using `std::size_t`, since `size_t` relies on `stddef.h` (or the wrapper `cstddef`) which is deprecated in C++11.